.gitignore

```diff
@@ -81,4 +81,5 @@ ivf_pq_index
 
 # cuvs_bench
 datasets/
-/*.json
\ No newline at end of file
+/*.json
+*.png
```

Comment:
I don't think we want to do a blanket ignore on png, especialy since we have png images in our docs. I'm okay ignoring json files, since those are assumed to be generated.